LCM/dsc/engine/ConfigurationManager/LocalConfigManagerHelper.c

```diff
@@ -507,6 +550,21 @@ MI_Result UnInitHandler(
         g_GetConfigFileName = NULL;
     }
 
+   if( g_InventoryFileName != NULL)
+    {
+        DSC_free(g_InventoryFileName);
+        g_InventoryFileName = NULL;
+    }
+
+<<<<<<< HEAD
+   if( g_InventoryReportFileName != NULL)
+    {
+        DSC_free(g_InventoryReportFileName);
+        g_InventoryReportFileName = NULL;
+    }
+
+=======
```

Comment:
That's really odd... This did not exist until I pushed my changes.  It simply wouldn't have compiled, and I rebuilt it from scratch tens of times before I pushed these changes.  Considering the other issues I was having with git rebase earlier, perhaps it is related.